edge-modules/TestAnalyzer/Controllers/ReportController.cs

```diff
@@ -0,0 +1,68 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace TestAnalyzer.Controllers
+{
+    using Microsoft.AspNetCore.Mvc;
+    using Microsoft.Azure.Devices.Edge.Util.AzureLogAnalytics;
+    using Newtonsoft.Json;
+
+    [Route("api/[controller]")]
+    [ApiController]
+    public class ReportController : Controller
+    {
+        // GET api/report/all
+        [HttpGet("all")]
+        public ActionResult<string> GetReport()
```

Comment:
Definitely it is better to return Task<T>; and please check if Task<JsonResult> is better than Task<ActionResult<string>>.

Reply:
I'm returning Task without ActionResult